samples/net/sockets/coap_server/README.rst

```diff
@@ -1,8 +1,8 @@
 .. zephyr:code-sample:: coap-server
-   :name: CoAP server
+   :name: CoAP service
    :relevant-api: coap udp
 
-   Use the CoAP library to implement a server that exposes CoAP resources.
+   Use the CoAP server subsystem that exposes CoAP resources.
```

Comment:
better wording below?

```suggestion
   Use the CoAP server subsystem to register CoAP resources.
```

Reply:
It feels like the rest of the README could be slightly reworded too and provide a brief explanation of how it now uses the CoAP service?